docs/next/guides/building-and-testing/modules.md

```diff
@@ -309,6 +665,27 @@ new Handsontable(container, {
 
 And that is all! You can use the numeric validator!
 
+#### Importing all validators at once
+
+You can quickly import and register all validators at once:
+
+```js
+// import all validators
+import {
+  registerValidator,
+  autocompleteValidator,
+  dateValidator,
+  numericValidator,
+  timeValidator,
+} from 'handsontable/validators';
+
+// import the `registerAllValidators()` function
+import { registerAllValidators } from 'handsontable/registry';
+
+// register all validators
+registerAllValidators();
+```
```

Comment:
```suggestion
```js
// import the `registerAllValidators()` function
import { registerAllValidators } from 'handsontable/registry';

// register all validators
registerAllValidators();
```
```